packages/web/src/services/AudiusBackend.ts

```diff
@@ -1724,6 +1837,20 @@ class AudiusBackend {
     hasWallet = false,
     referrer = null,
     feePayerOverride = null
+  }: {
+    email: string
```

Comment:
might be nice to create standalone type for this

Reply:
I agree but I think I'm gonna leave this for now because the type would have to live all the way at the top of the file outside of the AudiusBackend class. The benefit of having it colocated with the function outweighs the separate type imo here. Can def fix once AudiusBackend is split up and potentially not a class anymore